contracts/EasyAuction.sol

```diff
@@ -465,12 +466,14 @@ contract EasyAuction is Ownable {
             auctionData[auctionId].minFundingThresholdNotReached = true;
         }
         if (auctionData[auctionId].feeNumerator > 0) {
-            claimFees(auctionId);
+            claimFees(auctionId, fillVolumeOfAuctioneerOrder);
         }
-        claimAuctioneerFunds(auctionId);
-        (, uint96 priceNumerator, uint96 priceDenominator) =
-            clearingOrder.decodeOrder();
-        emit AuctionCleared(auctionId, priceNumerator, priceDenominator);
+        claimAuctioneerFunds(auctionId, fillVolumeOfAuctioneerOrder);
+        emit AuctionCleared(
+            auctionId,
+            fillVolumeOfAuctioneerOrder,
+            uint96(currentBidSum)
```

Comment:
This event logging should have the same price, though it has more information, as it displays the collected bidding amounts and the settled auctioningToken volume

Reply:
In principle this is different from the price if we need the exact amounts accounting for rounding. You could add `clearingOrder` to the logging to release the actual price (and extra information on the clearing price).